Week-5/symbols/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Unique Transaction IDs</title>
+    <script src="script.js" defer></script>
+</head>
+<body>
```

Comment:
The HTML is properly structured with the necessary script tag having the defer attribute, ensuring that the script is executed after the document has been parsed.